generateSecrets.js

```
@@ -0,0 +1,33 @@
+const fs = require('fs');
```

ESLint

[error] Extra semicolon.

(semi)